test/integration/patients/worklist/worklist.js

```diff
@@ -848,6 +1129,445 @@ context('worklist page', function() {
+        details: '',
+        due_date: testDate(),
+        due_time: '06:00:00',
+        duration: 10,
+        updated_at: testTsSubtract(4),
+        created_at: testTsSubtract(4),
+      },
+      relationships: {
+        state: getRelationship(stateTodo),
+        flow: getRelationship(testFlow),
+        files: getRelationship([]),
+        owner: getRelationship(getCurrentClinician()),
+        patient: getRelationship(testPatient1),
+        comments: getRelationship([]),
+      },
+    });
+
+    const testNewSocketAction = getAction({
+      attributes: {
+        name: 'Test Action - Created Elsewhere',
+        details: '',
+        due_date: testDate(),
+        due_time: '06:00:00',
+        duration: 10,
+        updated_at: testTsSubtract(3),
+        created_at: testTsSubtract(3),
+      },
+      relationships: {
+        state: getRelationship(stateTodo),
+        flow: getRelationship(testFlow),
+        files: getRelationship([]),
+        owner: getRelationship(getCurrentClinician()),
+        patient: getRelationship(testPatient1),
+        comments: getRelationship([]),
+      },
+    });
+
+    const testNewStateSocketAction = getAction({
+      attributes: {
+        name: 'Test Action - State Updated to Match Current Worklist Filter',
+        details: '',
+        due_date: testDate(),
+        due_time: '06:00:00',
+        duration: 10,
+        updated_at: testTsSubtract(2),
+        created_at: testTsSubtract(2),
+      },
+      relationships: {
+        state: getRelationship(stateTodo),
+        flow: getRelationship(testFlow),
+        files: getRelationship([]),
+        owner: getRelationship(getCurrentClinician()),
+        patient: getRelationship(testPatient1),
+        comments: getRelationship([]),
+      },
+    });
+
+    const testNewOwnerSocketAction = getAction({
+      attributes: {
+        name: 'Test Action - Owner Updated to Match Current Worklist Filter',
+        details: '',
+        due_date: testDate(),
+        due_time: '06:00:00',
+        duration: 10,
+        updated_at: testTsSubtract(1),
+        created_at: testTsSubtract(1),
+      },
+      relationships: {
+        state: getRelationship(stateTodo),
+        flow: getRelationship(testFlow),
+        files: getRelationship([]),
+        owner: getRelationship(getCurrentClinician()),
+        patient: getRelationship(testPatient1),
+        comments: getRelationship([]),
+      },
+    });
+
+    cy
+      .routesForPatientAction()
+      .routeActions(fx => {
+        fx.data = [testSocketAction];
+
+        fx.included.push(testPatient1, testFlow);
+
+        return fx;
+      })
+      .routePatient(fx => {
+        fx.data = testPatient1;
+
+        return fx;
+      })
+      .routeAction(fx => {
+        fx.data = testSocketAction;
+
+        return fx;
+      })
+      .routeFormByAction()
+      .routeFormDefinition()
+      .routeLatestFormResponse()
+      .visitOnClock('/worklist/owned-by', { now: testTs() })
```

Comment:
the visitOnClock is to set the time to now or why is this needed?

Reply:
It's to avoid flakiness when checking that the `updated_at` timestamp is updated correctly in the UI.

First introduced in this commit: https://github.com/RoundingWell/care-ops-frontend/commit/8eb7d817796c7852cc0ce7842fc676c38ae23017.